sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -126,8 +126,11 @@
       -XX:+CrashOnOutOfMemoryError
     </heapDumpOnOom>
 
+    <jacocoUnitTestAgent></jacocoUnitTestAgent>
```

Comment:
Why do we need this empty tag?

Reply:
Needed by Maven to inject the system property set by Jacoco when preparing the agent, same for the comment below